tox.ini

```diff
@@ -8,27 +8,21 @@ max-line-length = 99
 [testenv]
 deps =
     -rrequirements-test.txt
-    django18: Django~=1.8.0
-    django19: Django~=1.9.0
-    django110: Django~=1.10.0
     django111: Django~=1.11.0
 commands = python manage.py test tests/
 
-[testenv:flake8]
+[testenv:py37-flake8]
 skip_install = True
 deps = flake8
 commands = flake8 major_event_log tests setup.py
 
 [travis]
 python =
-    2.7: py27, flake8
-    3.4: py34
+    2.7: py27
     3.5: py35
     3.6: py36
+    3.7: py37, flake8
```

Comment:
So I would have "flake8" to be "py37-flake8", but it seems the correct env is running anyways. Does it still work correctly if we use "py37-flake8"?

Reply:
It still works fine when using `py37-flake8` .